libqtile/widget/launchbar.py

```diff
@@ -210,10 +213,11 @@ def draw(self):
                 # display an icon
                 self.drawer.ctx.set_source(self.surfaces[name])
                 self.drawer.ctx.paint()
-            self.drawer.draw(self.offset + xoffset, icon_width + self.padding)
+            self.drawer.draw(offsetx=self.offset + xoffset,
+                             width=icon_width + self.padding)
             xoffset += icon_width + self.padding
 
-    def calculate_width(self):
+    def calculate_length(self):
```

Comment:
Actually I've already used "size" to represent the other (minor) dimension of the bar, which previously was also referred to as "height". I think there's not much more we can do than this, I'll make sure to expand the comments describing the two terms.